spec/base_spec.rb

```diff
@@ -291,8 +291,6 @@ class DummyReplacedModel < DummyElasticSearchModel; end
       end
 
       it "validates the model before attempting to insert to Elasticsearch" do
-        expect { DummyElasticSearchModel.create! }.to raise_error(ActiveRecord::RecordInvalid, "Validation failed: My string must be set")
```

Comment:
Removing this expectation means the test is no longer testing anything and we're losing test coverage for the `required: true` functionality of class fields.

Is there a way we could update the class without removing the `required: true` and without changing this test?

Reply:
So yes, I'm sure there is a way we can do this, but I haven't found a straightforward way to do so that doesn't involve updating a lot of test cases. I've already spent a lot of time on this so if we could go over it together that'd be great.

Reply:
Looks like you really quickly found a way to make this work, nice 👍